src/main/java/com/futurecollars/OOPTask4/Convert.java

```diff
@@ -0,0 +1,14 @@
+package com.futurecollars.OOPTask4;
+
+import java.util.Scanner;
+
+public class Convert {
+    public static void convertion (double minutes) {
+
+        Scanner input = new Scanner(System.in);
+        minutes = input.nextDouble();
+        double seconds = minutes * 60;
+        System.out.println("That equals: " + seconds + "seconds");
```

Comment:
1. Po co Scanner skoro masz argument minutes?
2. Czemu wypisujesz, zamiast zwrócić? Przecież to jest aktualnie kompletnie nietestowalne